payments/utils.py

```diff
@@ -533,17 +533,23 @@ def get_notification_language(order):
 
 
 def get_email_subject(notification_type):
+    from applications.notifications import (
+        NotificationType as ApplicationsNotificationType,
+    )
+
     from .notifications import NotificationType
 
     if (
         notification_type == NotificationType.NEW_BERTH_ORDER_APPROVED
         or notification_type == NotificationType.RENEW_BERTH_ORDER_APPROVED
     ):
         return _("Boat berth invoice")
-    if notification_type == NotificationType.ORDER_CANCELLED:
+    elif notification_type == NotificationType.ORDER_CANCELLED:
         return _("Confirmation")
-    if notification_type == NotificationType.ORDER_REFUNDED:
+    elif notification_type == NotificationType.ORDER_REFUNDED:
         return _("Refund confirmation")
+    elif notification_type == ApplicationsNotificationType.BERTH_APPLICATION_REJECTED:
+        return _("Berth application processed")
     return notification_type.label
```

Comment:
This refactor is a bit out of scope, but since this is not only for payments, could you move it to `utils/messaging.py` or something similar?

Reply:
hmm yes, that would be a more proper place

Reply:
fixed

Reply:
Wonderful